apps/i18n/maze/en_us.json

```diff
@@ -60,13 +71,15 @@
   "noPathAhead": "path is blocked",
   "noPathLeft": "no path to the left",
   "noPathRight": "no path to the right",
-  "notAtFlowerError": "You can only get nectar from a flower.",
```

Comment:
This is still being used here: https://github.com/code-dot-org/code-dot-org/blob/3df48522322d984c3e347f61d93ed5f3b1117606/apps/src/maze/bee.js#L208